docs/interpolation.rst

```diff
@@ -0,0 +1,17 @@
+.. _interpolation:
+
+Interpolation of IRFs
+=====================
+
+This module contains functions to interpolate from a set of IRFs for different
+condiditions to a new IRF.
+
+This can e.\,g. be used to interpolate IRFs for zenith angles of 20° and 40°
```

Comment:
why the comma?

Reply:
Sorry, LaTeX muscle memory. This inserts a small non-breaking space in LaTeX, which is the proper way to typeset abbreviations.